src/lsg/patterns/icons/index.tsx

```diff
@@ -22,7 +22,7 @@ export interface IconProps {
 
 export enum Size {
 	XXS = 12,
-	XS = 18,
+	XS = 15,
```

Comment:
As I don't know all of the code, is this a visual breaking change? Was this size used beforehand?